atlp-rwanda/commanders-medica-backoffice
# Chore: project setup

#### What does this PR do?

This PR includes project setup with necessary packages and configurations

#### Description of Task to be completed?

- Add required packages
- Configure tailwind for styling
- Configure supabase as a backend
- Update Readme file with project description

#### How should this be manually tested?

Run `npm run dev` after installing packages and necessary environment variables. Check that the app starts without errors.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker/Trello stories?

N/A

#### Screenshots (if appropriate)

N/A

#### Questions:

N/A
